### YamlMime:UniversalReference
ms.openlocfilehash: a43fb1f930b44c609c80f919f0c41628cce43c2c
ms.sourcegitcommit: efa2d98deffe8a0d41a8d63f9f07aa720862e6ab
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 12/13/2018
ms.locfileid: "52125068"
items:
- uid: azure-iot-common.ArgumentError
  name: ArgumentError
  fullName: ArgumentError
  children:
  - azure-iot-common.ArgumentError.constructor
  - azure-iot-common.ArgumentError.Error
  langs:
  - typeScript
  type: class
  summary: 인수가 유효 하지 않을 때 발생 하는 데 오류가 발생 했습니다.
  extends:
    name: Error
  package: azure-iot-common
- uid: azure-iot-common.ArgumentError.constructor
  name: ArgumentError(string)
  children: []
  type: constructor
  langs:
  - typeScript
  summary: ''
  syntax:
    content: 'new ArgumentError(message?: string)'
    parameters:
    - id: message
      type:
      - string
      description: ''
      optional: true
  package: azure-iot-common
- uid: azure-iot-common.ArgumentError.Error
  name: Error
  fullName: Error
  children: []
  langs:
  - typeScript
  type: property
  summary: ''
  syntax:
    content: 'static Error: ErrorConstructor'
    return:
      type:
      - ErrorConstructor
  package: azure-iot-common
